js/components/grid/calendar.js.jsx

```diff
@@ -334,7 +334,17 @@ function initializeSessions(lectures) {
  */
 function setWidths(session) {
   let days = ['M', 'T', 'W', 'R', 'F'];
-
+  // Reset all widths back to 1 and inConflict to false
+  for (let i = 8; i < 22; i++) {
+    let timeRow = session[i];
+    days.forEach(function(day) {
```

Comment:
You're using the ES6 function syntax already, so switch this one to use that syntax as well. (Contrast lines 340 and 342). (This occurs a couple of other times in this PR as well)